.gitignore

```diff
@@ -1 +1,2 @@
 target
+.idea
```

Comment:
Btw you best save yourself the hassle and ignore this globally instead fo repo-by-repo.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
@dwijnand You are right. Actually, I do it often for other files like `.zshrc` that I usually have per project. Somehow, I keed adding `.idea` to `.gitignore`. 

That will be the last time. ;-)